### PR TITLE
Drop obsolete python and django versions from test suites

### PR DESCRIPTION
**Pull request by dmitry-mukhin**
- separate django tests
- refactor tox and travis config
- drop obsolete python and django versions from tests

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,59 +1,80 @@
+# https://devguide.python.org/#branchstatus
 # https://www.djangoproject.com/download/#supported-versions
+# https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django
```
Comment (andrewshkovskii) [inline .travis.yml]: 👍

File: README.rst [hunk 2]
```diff
@@ -66,7 +66,7 @@ Features
 Requirements
 ------------
 
-``pyuploadcare`` requires Python 2.6, 2.7, 3.3, 3.4, 3.5 or 3.6
+``pyuploadcare`` requires Python 2.7, 3.4, 3.5 or 3.6
```
Comment (andrewshkovskii) [inline README.rst]: In here can we say, that `pyuploadcare` does not tested with other versions of python, but if user want - he can try to use other versions?

File: setup.py [hunk 3]
```diff
@@ -13,13 +13,12 @@
     'six>=1.2.0',
 ]
 
-if PY_VERSION == (2, 6):
-    requirements.append('argparse')
-
 if PY_VERSION < (3, 0):
-    long_description = open('README.rst').read() + '\n\n' + open('HISTORY.rst').read()
+    long_description = open('README.rst').read() + '\n\n' + \
```
Comment (andrewshkovskii) [inline setup.py]: Better to use `(`, `)` for multi-line statements than `\`

File: README.rst (hunk 2 above)
Comment (dmitry-mukhin) [reply README.rst]: yup, you missed the conversation yesterday.
I agree that we can state that it should work for quite some time with old version and offered to create an extended tox config.

but. officially we're dropping support for obsolete, so `setup.py` should reflect that.

File: setup.py (hunk 3 above)
Comment (dmitry-mukhin) [reply setup.py]: agreed

**Comment by dmitry-mukhin** [comment]
travis config reflects only officially supported versions to conserve build time. tox run is much much faster.

**Comment by dmitry-mukhin** [comment]
bump!
@andrewshkovskii @homm

**Pull request merged**